src/test/java/org/fcrepo/migration/Example1TestSuite.java

```diff
@@ -180,20 +181,24 @@ public void testDS4ExternalParsing() throws IOException {
         Assert.assertEquals("DS4.0", ds4.getVersionId());
         Assert.assertEquals("Example External datastream.", ds4.getLabel());
         Assert.assertEquals("2015-01-27T19:14:38.999Z", ds4.getCreated());
-        Assert.assertEquals("text/html", ds4.getMimeType());
```

Comment:
The magic number: `dsVersions.get(6)` seems a bit fragile, above.
Likewise with the other test methods in this class.